.env.example

```
@@ -0,0 +1,17 @@
+DEFAULT_LLM_PROVIDER=gemini
+FALLBACK_LLM_PROVIDER=anthropic
+
+# Anthropic Claude configuration
+ANTHROPIC_API_KEY=your_anthropic_api_key_here
+CLAUDE_MODEL=claude-3-haiku-20240307
+FALLBACK_CLAUDE_MODEL=claude-3-5-sonnet-20241022
+
+# Google Gemini configuration
+GEMINI_API_KEY=your_gemini_api_key_here
+GEMINI_MODEL=gemini-1.5-flash
+FALLBACK_GEMINI_MODEL=gemini-1.5-pro
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)